data_source_obmcs_database_db_system.go

```diff
@@ -101,15 +101,19 @@ func (s *DBSystemDatasourceCrud) SetData() {
 				"compartment_id":          r.CompartmentID,
 				"cpu_core_count":          int(r.CPUCoreCount),
 				"data_storage_percentage": r.DataStoragePercentage,
+				"data_storage_size_in_gb": int(r.DataStorageSizeInGB),
 				"database_edition":        r.DatabaseEdition,
 				"db_home":                 []interface{}{dbHome},
 				"display_name":            r.DisplayName,
 				"disk_redundancy":         r.DiskRedundancy,
 				"domain":                  r.Domain,
 				"hostname":                r.Hostname,
 				"id":                      r.ID,
+				"license_model":           r.LicenseModel,
 				"lifecycle_details":       r.LifecycleDetails,
 				"listener_port":           int(r.ListenerPort),
+				"node_count":              int(r.NodeCount),
+				"reco_storage_size_in_gb": int(r.RecoStorageSizeInGB),
```

Comment:
Is this in the resource as well?

Reply:
"node_count" is in the resource, but "reco_storage_size_in_gb" is not.  According to their API spec, "reco_storage_size_in_gb" looks like a derived (computed) attribute, i.e, the request doesn't contain the attribute.

Reply:
Excuse me, I now understand what you mean behind the question.  Will add "reco_storage_size_in_gb" to the schema.  Thanks.